test/models/audio_file_test.rb

```diff
@@ -90,4 +90,15 @@ def setup
       end
     end
   end
+
+  test "should touch the associated track when it's destroyed" do
+    one_day_ago = 1.day.ago
+    track = Track.create(number: 0, title: 'title', album: Album.create(title: 'title'), audio_file: @audio_file, updated_at: one_day_ago)
+
+    assert_equal one_day_ago, track.updated_at
+
+    @audio_file.destroy
+
+    assert_not_equal one_day_ago, track.reload.updated_at
```

Comment:
```suggestion
    assert_changes 'track.reload.updated_at', from: one_day_ago do
      @audio_file.destroy!
    end
```

Maybe using `assert_changes` is a clearer expression of what we expect to happen here?
I'd also use `destroy!` so this blows up if the file can't be deleted